docker-app/qfieldcloud/core/views/projects_views.py

```diff
@@ -102,7 +102,7 @@ class ProjectViewSet(viewsets.ModelViewSet):
     serializer_class = ProjectSerializer
     lookup_url_kwarg = "projectid"
     permission_classes = [permissions.IsAuthenticated, ProjectViewSetPermissions]
-    pagination_class = pagination.QfcLimitOffsetPagination
+    pagination_class = pagination.QfcLimitOffsetPagination(count_entries=True)
```

Comment:
```suggestion
    pagination_class = pagination.QfcLimitOffsetPagination()
```

Make `count_entries` should be `True` for now by default. Also assume it is always not-None.